packages/core/src/requestforattestation/RequestForAttestation.ts

```diff
@@ -206,7 +206,7 @@ export default class RequestForAttestation implements IRequestForAttestation {
    * ```
    */
   public removeClaimOwner(): void {
-    delete this.claim.owner
+    delete (this.claim as Partial<IClaim>).owner
```

Comment:
kinda hacky (though I know that the deletion doesn't fit with the defined type of `this.claim`). What about the `@ts` comments you did in the test files. Is there one, where we can say: "just let it be"?

Reply:
it was `@ts-ignore` before; if you like that better, I can revert. We have a linter rule against its usage though, so we need to escape that as well.

Reply:
I would rather want to have a warning for it.
So if the linting is configured to show it as an error, let's reconfigure it to list it as warning.
Because it is actually not the correct way to do it and we will need the reminder.